docs/install/windows.md

```diff
@@ -1,37 +1,407 @@
+
+```
+./mingw64.exe
+```
+ or `/mnt/c/msys64$ ./mingw64.exe `
+ 
+It will open a terminal. It is not PowerShell, it is not CMD, it is a mingw64 teminal.
+ 
+------------------------------------------------------------ 
+## How to build NEURON
+
+### Step 12: Creating the exe
+
+Now, inside this terminal (it is a mingw64 terminal, see step above) go to `C:\Users\User\Neuron\nrn\ci` and run 
+
+```
+./win_build_cmake.sh 
+```
+ 
+It will create an exe file `nrn-nightly-AMD64` (at `C:\Users\User\Neuron\nrn\nrn-nightly-AMD64`).
+ 
+------------------------------------------------------------ 
+
+### Step 13: Running the exe
+
+Double click at `nrn-nightly-AMD64` and it will install NEURON. 
+It is a standard installation. 
+
+That means the files created at `C:\Users\User\Neuron\nrn\build\bin` will be coped to:  `C:\nrn\bin` and the "installation" will appear at `C:\nrn` and `C:\nrn-install` 
+ 
+------------------------------------------------------------ 
+
+
+### Step 14: Using and modifying NEURON file... 
+You can replace any new `.dll` file at `C:\nrn\bin` and the NEURON will present the new feature implemented. 
+ 
+Example: 
+
+#### Step 14.A: Checking the `nrngui` 
+Run `nrngui` (inside the folder `C:\Users\User\Desktop\NEURON 9.0.0 AMD64` double click on `nrngui`)
+
+It will show something like:
+
 ```
-This will download all required installers and corresponding versions(`Python`, `MPI`, `nsis`)
+NEURON -- VERSION X.X.X 2024-07-09
+
+Duke, Yale, and the BlueBrain Project -- Copyright 1984-2022
+See http://neuron.yale.edu/neuron/credits
+
+oc>
+```
+
+Close this NEURON window.
+
+#### Step 14.B: Making some changes at `init.cpp`
+
+Open the `init.cpp` file (at `C:\Users\User\Neuron\nrn\src\nrnoc\init.cpp`  or at `C:\msys64\home\User\Neuron\nrn\src\nrnoc\init.cpp`) look for `hoc_last_init(void)` and the follow code inside it:
+
+```C
+    if (nrnmpi_myid < 1)
+        if (nrn_nobanner_ == 0) {
+            Fprintf(stderr, "%s\n", nrn_version(1));
+            Fprintf(stderr, "%s\n", banner);
+            IGNORE(fflush(stderr));
+        }
+```
+
+
+modifying this introduction the print line `printf("\n Now It is my NEURON version!!!! \n\n");` for example: 
+
+```C
+     if (nrnmpi_myid < 1)
+        if (nrn_nobanner_ == 0) {
+            Fprintf(stderr, "%s\n", nrn_version(1));
+            printf("\nNow It is my NEURON version!!!! \n\n");
+            Fprintf(stderr, "%s\n", banner);
+            IGNORE(fflush(stderr));
+        }
+```
+and save.
+
+#### Step 14.C: Compiling the new code.
+
+Compile this new code. We can compile using Step 15. (Do it and come back here) 
+
+#### Step 14.D: Replacing the `libnrniv.dll` file
+
+It will create a new `libnrniv.dll` (and `libnrnmpi_msmpi.dll` and a new `libnrnpython3.XX.dll` and new `nrniv`, but just the first one matters for THIS example) at `nrn\build\bin` (at `C:\Users\User\Neuron\nrn\build\bin` or at `C:\msys64\home\User\Neuron\nrn\build\bin` if you did the installation process here). 
```

Comment:
I'd really avoid giving them multiple options for how to do things. Maybe just pick one and use it for the entire example.

Also, instead of "It", how about "The compilation step updates `libnrniv.dll` and some other files, but only `libnrniv.dll` nmatters for this example. The updated file is at..."